Index: Makefile.in
===================================================================
--- Makefile.in	(revision 149)
+++ Makefile.in	(working copy)
@@ -18,10 +18,10 @@
 #   along with Ragel; if not, write to the Free Software
 #   Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA 
 
+BUILD_COMMON = common redfsm
+BUILD_SUBDIRS = ragel rlgen-cd rlgen-java rlgen-ruby rlgen-dot
+ALL_SUBDIRS = $(BUILD_COMMON) $(BUILD_SUBDIRS) test examples doc
 
-BUILD_SUBDIRS = common ragel redfsm rlgen-cd rlgen-java rlgen-ruby rlgen-dot
-ALL_SUBDIRS = $(BUILD_SUBDIRS) test examples doc
-
 #*************************************
 
 # Programs
@@ -33,11 +33,14 @@
 # build targets
 all: $(BUILD_SUBDIRS)
 
-.PHONY: $(BUILD_SUBDIRS)
+.PHONY: $(ALL_SUBDIRS)
 
-$(BUILD_SUBDIRS):
+$(BUILD_COMMON):
 	@cd $@ && $(MAKE)
 
+$(BUILD_SUBDIRS): $(BUILD_COMMON)
+	cd $@ && $(MAKE)
+
 # clean targets.
 
 CLEAN_SUBDIRS = $(ALL_SUBDIRS:%=%-clean)
Index: doc/Makefile.in
===================================================================
--- doc/Makefile.in	(revision 149)
+++ doc/Makefile.in	(working copy)
@@ -40,6 +40,8 @@
 
 ragel-guide.pdf: $(PDFFIGS) $(INPUT)
 
+man: $(MANPAGES)
+
 %.pdf: %.fig
 	fig2dev -L pdf $< $@
 
