diff -rdu ragel-5.22.orig/doc/ragel.1.in ragel-5.22/doc/ragel.1.in
--- ragel-5.22.orig/doc/ragel.1.in	2007-06-14 08:10:18.000000000 -0600
+++ ragel-5.22/doc/ragel.1.in	2007-06-18 20:53:54.000000000 -0600
@@ -31,7 +31,8 @@
 .SH DESCRIPTION
 .B Note:
 this is the frontend component of Ragel, which generates an intermediate
-file format that must be processed by rlcodegen(1).
+file format that must be processed by one of: rlgen-cd(1), rlgen-dot(1),
+rlgen-java(1).
 
 Ragel compiles finite state machines from regular languages into executable
 code.  Ragel can generate C, C++, Objective-C, D, or Java code. Ragel state
diff -rdu ragel-5.22.orig/README ragel-5.22/README
--- ragel-5.22.orig/README	2007-04-06 12:25:12.000000000 -0600
+++ ragel-5.22/README	2007-06-18 20:50:53.000000000 -0600
@@ -33,7 +33,7 @@
 
 To build all the documentation cd to 'doc' and type 'make'. If you don't have
 all of the programs to build the user guide and just want the man page use
-'make ragel.1 rlcodegen.1'.
+'make ragel.1 rlgen-cd.1 rlgen-java.1 rlgen-ruby.1 rlgen-dot.1'
 
 
 3. Installing
Only in ragel-5.22.orig/redfsm: redfsm.a
